packages/snaps-cli/scripts/computeSnapShasum.js

```diff
@@ -1,4 +1,5 @@
 const { readFileSync } = require('fs');
```

Comment:
I think we can remove this script and its dependencies entirely?

Reply:
It's still used in the `shasum` script (`package.json`). Not sure if that's used anywhere.

Reply:
I don't think so, but we can remove at another time